src/components/layout-components/header.tsx

```diff
@@ -106,22 +116,25 @@ const Header: React.FC = ({
       label={ strings.header.user }
       value={ userEmail }
       id="user-email"
-      renderValue={ () => userEmail}
+      renderValue={ () => renderSelectValue(userEmail) }
     >
       <MenuItem style={{ backgroundColor: "#fff" }}>{ renderLogoutButton() }</MenuItem>
     </HeaderSelect>
   );
 
   return (
-    <Root position="static">
+    <Root
+      position="static"
+      sx={{ backgroundColor: process.env.REACT_APP_THEME_USE_COLORED_HEADER === "true" ? process.env.REACT_APP_THEME_PALETTE_SECONDARY : "" }}
```

Comment:
Should this logic also be in theme file. Starts to look bit messy if it's used in the component.

Reply:
Same question for the rest of the file